lib/betagouvbot.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 # frozen_string_literal: true
 
-Dir[__dir__ + '/betagouvbot/**/*.rb'].sort.each(&method(:require))
+Dir[__dir__ + '/betagouvbot/**/*.rb'].each(&method(:require))
```

Comment:
My gut objects to this very strongly. We found out that the behavior of `Dir.each` is platform-dependent, "thanks" to production downtime. The sort was introduced to restore deterministic behavior here. Taking it out feels like playing Russian Roulette with the next similar problem we introduce without our tests catching it owing to unlucky coincidence.

Now that I've had a chance to examine this line, I also find myself objecting to it, I'd prefer explicit (and minimal) statements of dependencies, a habit from my Elm and Haskell forays.